components/filter/demo/filter-overflow-group.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script type="module">
+		import '../../demo/demo-page.js';
+		import '../../filter/filter.js';
+		import '../../filter/filter-dimension-set.js';
+		import '../../filter/filter-dimension-set-value.js';
+		import '../../filter/filter-overflow-group.js';
+		import '../../filter/filter-tags.js';
+	</script>
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1.0">
+    <meta charset="UTF-8">
+	<style>
+		d2l-filter-tags {
+			padding-top: 1rem;
+		}
+	</style>
+</head>
+
+<body unresolved>
+
+	<d2l-demo-page page-title="d2l-filter-overflow-group">
+
+		<h2>Filter Overflow Group</h2>
+		<d2l-demo-snippet>
+			<template>
+				<d2l-filter-overflow-group>
+					<d2l-filter>
+						<d2l-filter-dimension-set key="skill" text="Skill">
+							<d2l-filter-dimension-set-value key="communication" text="Fall"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="leadership" text="Winter"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="management" text="Spring"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="planning" text="Summer"></d2l-filter-dimension-set-value>
+						</d2l-filter-dimension-set>
+					</d2l-filter>
+					<d2l-filter>
+						<d2l-filter-dimension-set key="type" text="Type" selection-single>
+							<d2l-filter-dimension-set-value key="certificate" text="Certificate"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="degree" text="Degree"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="diploma" text="Diploma"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="course" text="Course"></d2l-filter-dimension-set-value>
+						</d2l-filter-dimension-set>
+					</d2l-filter>
+					<d2l-filter>
+						<d2l-filter-dimension-set key="provider" text="Semester3">
+							<d2l-filter-dimension-set-value key="mcmaster" text="McMaster"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="powered" text="PowerED"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="guelph" text="University of Guelph"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="manitoba" text="University of Manitoba"></d2l-filter-dimension-set-value>
+						</d2l-filter-dimension-set>
+					</d2l-filter>
+					<d2l-filter>
+						<d2l-filter-dimension-set key="format" text="Format">
+							<d2l-filter-dimension-set-value key="selfpaced" text="Self-Paced"></d2l-filter-dimension-set-value>
+							<d2l-filter-dimension-set-value key="instructor" text="Instructor Lead" selected></d2l-filter-dimension-set-value>
+						</d2l-filter-dimension-set>
+					</d2l-filter>
+					<d2l-filter>
+						<d2l-filter-dimension-set key="language" text="Language" select-single>
```

Comment:
Should this be `selection-single` or is this a new thing I missed in a previous review?